packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -204,6 +204,20 @@ export const PostsListWithVotesAndSequence = () => frag`
   }
 `
 
+export const UltraFeedPostFragment = () => frag`
+  fragment UltraFeedPostFragment on Post {
+    ${PostsListWithVotes}
+    contents {
+      _id
+      html
```

Comment:
We're now fetching the entire post html for all the posts loaded in the feed?

Reply:
@b0b3rt Yes. I suppose an alternative would be to do a useSingle upon first click/expansion?

Reply:
That's probably better, not 100% sure

Reply:
I'm down to make the change but would rather not go do it before shipping this